src/uikit.cs

```diff
@@ -5482,7 +5482,20 @@ partial interface UIFontDescriptor : NSSecureCoding, NSCopying {
 
 		[Export ("fontDescriptorWithSymbolicTraits:")]
 		UIFontDescriptor CreateWithTraits (UIFontDescriptorSymbolicTraits symbolicTraits);
-		
+
+		[NoiOS][NoTV]
+		[Watch (5,2)]
+		[Export ("fontDescriptorWithDesign:")]
+		[return: NullAllowed]
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		UIFontDescriptor CreateWithDesign (NSString design);
```

Comment:
Any reason for not using `BindAs` here?

```csharp
UIFontDescriptor CreateWithDesign ([BindAs (typeof (UIFontDescriptorSystemDesign))] NSString design);
```

Reply:
That would _hide_ the `NSString` overload, right ?